x-pack/plugins/monitoring/server/alerts/disk_usage_alert.ts

```diff
@@ -96,19 +100,24 @@ export class DiskUsageAlert extends BaseAlert {
   }
 
   protected getDefaultAlertState(cluster: AlertCluster, item: AlertData): AlertState {
-    const currentState = super.getDefaultAlertState(cluster, item);
-    currentState.ui.severity = AlertSeverity.Warning;
-    return currentState;
+    const base = super.getDefaultAlertState(cluster, item);
+    return {
+      ...base,
+      ui: {
+        ...base.ui,
+        severity: AlertSeverity.Warning,
+      },
+    };
   }
```

Comment:
nit: I don't think this method should be changed. The end result is the same, but this makes it more confusing (imo). And, same goes for `getDefaultAlertState` in `memory_usage_alert.ts`

Reply:
changed back